packages/cloud_security_posture/kibana/index_pattern/cloud_security_posture-9129a080-7f48-11ec-8249-431333f83c5f.json

```
@@ -0,0 +1,14 @@
+{
+    "attributes": {
+        "description": "",
+        "title": "logs-cloud_security_posture.findings-*"
```

```suggestion
        "title": "logs-cloud_security_posture.findings*"
```
@kfirpeled I know it's already merged, but since the current findings index is `logs-cloud_security_posture.findings_latest-default` this data view not catch it 😢

---

@oren-zohar This is on purpose, we'll move the `findings_latest` index in a different PR